### ISSUE TITLE
await View3D1.SetCameraPositionAsync(new Vector3(0, 100, 250));  is Broken

### ISSUE THREAD

**Issue by Hypermobile12** [simutaroman/Blazor3D#3]
on Example02

```
   private async Task OnLoadStlButtonClick()
    {
        //todo: set custom material
        await View3D1.Import3DModelAsync(
        Import3DFormats.Stl,
        "https://threejs.org/examples/models/stl/ascii/slotted_disk.stl", null);
        await View3D1.SetCameraPositionAsync(new Vector3(0, 1, 1));
    }
```
        
    
The _SetCameraPositionAsync_ is Broken on V0.1.12

just Click LoadSTL, and you'll get the error

```
warn: Microsoft.AspNetCore.Components.Server.Circuits.RemoteRenderer[100]
      Unhandled exception rendering component: Lu.setCameraPosition is not a function
TypeError: Lu.setCameraPosition is not a function
    at Module.Pu (https://localhost:7264/_content/Blazor3D/js/bundle.js:2:701389)
    at https://localhost:7264/_framework/blazor.server.js:1:3501
```

etc.

**Comment by simutaroman** [comment]
you're right, thanks. I'll fix it in the next release

**Comment by simutaroman** [comment]
fixed since v0.1.13.
notice, that the method now has the next signature Viewer.SetCameraPositionAsync(Vector3 position, Vector3 lookAt));
